packages/threat-composer/src/customTypes/mitigations.ts

```diff
@@ -32,3 +38,11 @@ export const MitigationLinkSchema = EntityLinkBaseSchema.extend({
 }).strict();;
 
 export type MitigationLink = z.infer<typeof MitigationLinkSchema>;
+
+export interface MitigationListFilter {
+  status?: (typeof MITIGATION_STATUS_IDENTIFIED
+    | typeof MITIGATION_STATUS_IN_PROGRESS
+    | typeof MITIGATION_STATUS_RESOLVED
+    | typeof MITIGATION_STATUS_RESOLVED_ABANDONED
```

Comment:
Upon reflection, 'Abandoned' carries a negative connotation - and that it is simply left behind or neglected. I propose we change to 'Will not action' which carries slightly less negative connotation in my opinion, and is clear that it's a _decision_:

`MITIGATION_STATUS_RESOLVED_WILLNOTACTION`